staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -251,6 +251,12 @@ type Config struct {
 	// rejected with a 429 status code and a 'Retry-After' response.
 	ShutdownSendRetryAfter bool
 
+	// AnnotateEarlyAndLateRequests indicates whether the request(s) that
+	// arrive too early (before the apiserver is ready to serve) or late
+	// (during the server shutdown window) should be annotated so they
+	// can be identified by inspecting the apiserver audit logs.
+	AnnotateEarlyAndLateRequests bool
```

Comment:
why would someone choose to disable this.

Reply:
I thought it would be more palatable to upstream acceptance - by default it will be turned off. (If we don't get it accepted, then we can drop the configuration)